index.js

```diff
@@ -3,6 +3,11 @@ const prompter = require('./src/prompter');
 const statusManager = require('./src/statusManager');
 const argv = require('./src/arguments')
 
+function countVulnerabilities(advisories, severity) {
+    return Object.values(advisories)
```

Comment:
I'm trying to keep support for node4, so this would have to be `Object.keys.map` or something similar.